operators/jaeger/1.26.0/jaeger.v1.26.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,515 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "jaegertracing.io/v1",
+          "kind": "Jaeger",
+          "metadata": {
+            "name": "jaeger-all-in-one-inmemory"
+          }
+        }
+      ]
+    capabilities: Deep Insights
+    categories: Logging & Tracing
+    certified: "false"
+    containerImage: docker.io/jaegertracing/jaeger-operator:1.21.2
+    createdAt: "2019-09-04T13:28:40+00:00"
+    description: Provides tracing, monitoring and troubleshooting for microservices-based
+      distributed systems
+    operators.openshift.io/infrastructure-features: '["disconnected"]'
+    repository: https://github.com/jaegertracing/jaeger-operator
+    support: Jaeger Community
+  name: jaeger-operator.v1.26.0
```

Comment:
Recent versions are available for multiple architectures (but see below!); this is needed to have them show in the OpenShift web console OperatorHub on non-x86:
```
  labels:
    operatorframework.io/arch.amd64: supported
    operatorframework.io/arch.arm64: supported
    operatorframework.io/arch.s390x: supported
```